### PR TITLE
docs(api): document 2024.3 API deprecation and replacement

### PR DESCRIPTION
**Pull request by educhastenier**
relates to https://bonitasoft.atlassian.net/browse/BPM-242

### CODE REVIEW

File: modules/identity/pages/admin-api-permissions.adoc [hunk 1]
```diff
@@ -289,4 +289,4 @@ These are additional REST APIs that you have access to when you are using the Su
 
 === Additional accessible APIs
 * GET
-** platform/license
+** system/information
```
Comment (educhastenier) [inline modules/identity/pages/admin-api-permissions.adoc:292]: @abirembaut am I wrong with this change?

File: modules/identity/pages/super-admin-api-permissions.adoc [hunk 2]
```diff
@@ -180,4 +180,4 @@ These are additional REST APIs that you have access to when you are using the Su
 
 === Additional accessible APIs
 * GET
-** platform/license
+** system/information
```
Comment (educhastenier) [inline modules/identity/pages/super-admin-api-permissions.adoc:183]: @abirembaut am I wrong with this change?

Comment (educhastenier) [reply modules/identity/pages/super-admin-api-permissions.adoc:183]: @abirembaut up please

Comment (abirembaut) [reply modules/identity/pages/super-admin-api-permissions.adoc:183]: :heavy_check_mark:

File: modules/identity/pages/admin-api-permissions.adoc (hunk 1 above)
Comment (abirembaut) [reply modules/identity/pages/admin-api-permissions.adoc:292]: :heavy_check_mark:

**Review by abirembaut — APPROVED** [review]

**Pull request merged**